shared/src/packages/orders/libs/types/order-status.type.ts

```diff
@@ -0,0 +1,5 @@
+import { type OrderStatus } from '../enums/enums.js';
+
+type OrderStatus = (typeof OrderStatus)[keyof typeof OrderStatus];
```

Comment:
Just use ValueOf<typeof OrderStatus> when needed

Reply:
Don't noticed it before, used ValueOf as suggested